### PR TITLE
Adding repeat pending button

### PR DESCRIPTION
**Pull request by flbulgarelli**
<del>This solution is quick and dirty.

There is another branch for refactoring and generalizing the navigation model and views of guides and exercises
</del>

Fixes #303

### CODE REVIEW

**Comment by flbulgarelli** [comment]
screencapture-localhost-3000-exercises-funcional-haskell-7-los-operadores-son-funciones-1441811944328

**Comment by flbulgarelli** [comment]
@faloi Please review and merge

File: config/locales/es.yml [hunk 1]
```diff
@@ -170,3 +170,4 @@ es:
   editor: "Editor"
   console: "Consola"
   type_expressions_here: "Acá podés escribir expresiones para probar tu solución"
+  repeat_pending: "Terminar pendientes"
```
Comment (faloi) [inline config/locales/es.yml]: I think it's more like _next_pending_exercise_, so I would say "Siguiente ejercicio pendiente"

File: app/helpers/concerns/with_navigation.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module WithNavigation
-  def next_button(navigable)
-    sibling_button(navigable.next_for(current_user), :next_exercise, 'chevron-right', 'btn btn-success', true)
+  def next_button(navigable) #TODO duplicated logic with next_guides_box
+    sibling_button(navigable.next_for(current_user), :next_exercise, 'chevron-right', 'btn btn-success', true) ||
```
Comment (faloi) [inline app/helpers/concerns/with_navigation.rb]: :see_no_evil:, speciallly the last boolean parameter

Comment (flbulgarelli) [inline app/helpers/concerns/with_navigation.rb]: Yeah, yeah. I said that a little refactor is comming soon :smile:

File: config/locales/es.yml (hunk 1 above)
Comment (flbulgarelli) [inline config/locales/es.yml]: Ok, maybe you are right. 

We are using here [repeat](http://fortawesome.github.io/Font-Awesome/icon/repeat/) icon. What about simply using [next icon](http://fortawesome.github.io/Font-Awesome/icon/chevron-right/) like the rest?

Comment (faloi) [inline config/locales/es.yml]: Ok, the different color and caption are representative enough, and the
arrow fits in better.

File: app/models/concerns/with_exercises.rb [hunk 3]
```diff
@@ -11,7 +11,6 @@ def exercises_count
 
   def pending_exercises(user)
     exercises.
-        at_locale.
```
Comment (flbulgarelli) [inline app/models/concerns/with_exercises.rb]: This is not necessary anymore.

**Comment by flbulgarelli** [comment]
@faloi I hope you are happy now :stuck_out_tongue: 

Please review and merge

**Comment by faloi** [comment]
You could give `self.class.name.downcase.pluralize` a name.

(Sorry, GitHub mobile doesn't support inline comments)

**Pull request merged**